guides/v2.1/architecture/gdpr/magento-2x.md

```diff
@@ -59,7 +59,7 @@ Magento 2 stores these customer attributes:
 
 #### `customer_entity` and reference tables
 
```

Comment:
Remove changes to this file from the PR. This is GDPR compliance and we need to control it.

Reply:
Removed changes